### PR TITLE
configurable mq ports

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

File: netclient/functions/daemon.go [hunk 1]
```diff
@@ -209,7 +210,7 @@ func NewTLSConfig(server string) *tls.Config {
 func setupMQTT(cfg *config.ClientConfig, publish bool) (mqtt.Client, error) {
 	opts := mqtt.NewClientOptions()
 	server := cfg.Server.Server
-	opts.AddBroker("ssl://" + server + ":8883") // TODO get the appropriate port of the comms mq server
+	opts.AddBroker("ssl://" + server + servercfg.GetMQPort())
```
Comment (0xdcarns) [inline netclient/functions/daemon.go:213]: are you missing a ":" here?

File: servercfg/serverconf.go [hunk 2]
```diff
@@ -214,7 +225,7 @@ func GetMessageQueueEndpoint() string {
 		host = config.Config.Server.MQHOST
 	}
 	//Do we want MQ port configurable???
-	return host + ":1883"
+	return host + GetMQServerPort()
```
Comment (0xdcarns) [inline servercfg/serverconf.go:228]: are you missing a ":" here?

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

File: servercfg/serverconf.go (hunk 2 above)
Comment (afeiszli) [reply servercfg/serverconf.go:228]: shouldn't have ":" server side

File: netclient/functions/daemon.go (hunk 1 above)
Comment (afeiszli) [reply netclient/functions/daemon.go:213]: yes, need to add

Comment (mattkasun) [reply netclient/functions/daemon.go]: good catch

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**